docs/files/grafana-dashboard.json

```diff
@@ -736,6 +737,18 @@
           "legendFormat": "Non-Heap",
           "range": true,
           "refId": "B"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "expr": "sum(jvm_memory_max_bytes{instance=\"$instance\", area=\"heap\"})",
+          "hide": false,
+          "legendFormat": "__auto",
```

Comment:
Can we have a label for this one? I believe it's the `legendFormat` field, it's set to `Heap` and `Non-Heap` for the other two fields. Maybe call this one `Total` or so, to make it a bit more readable.

Reply:
good suggestion, added now